nemo/collections/asr/models/hybrid_rnnt_ctc_models.py

```diff
@@ -339,6 +348,7 @@ def change_decoding_strategy(self, decoding_cfg: DictConfig = None, decoder_type
         decoding_cfg = OmegaConf.merge(decoding_cls, decoding_cfg)
 
         self.ctc_decoding = CTCDecoding(decoding_cfg=decoding_cfg, vocabulary=self.ctc_decoder.vocabulary)
+        self.decoding = self.ctc_decoding
```

Comment:
I don't think you need to do it here. It would overwrite the RNNT decoding and we cannot switch back. There are other places in the code which are not based on this assumption.

Reply:
removed